apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+/* global ApplicationSource */
+/* global BookmarkSource */
+/* global CollectionSource */
```

Comment:
Are we doing collections and bookmarks for this version of the homescreen, or can these be cut?